pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -22,10 +22,15 @@ const (
 func checkVersionOfWindowsUpdate() (bool, error) {
 	windowsReleaseId := `(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId`
 
-	stdOut, _, _ := powershell.Execute(windowsReleaseId)
+	stdOut, _, err := powershell.Execute(windowsReleaseId)
+	if err != nil {
+		logging.Debug(err.Error())
+		return false, errors.New("Failed to get Windows release id")
+	}
 	yourWindowsReleaseId, err := strconv.Atoi(strings.TrimSpace(stdOut))
 
 	if err != nil {
+		logging.Debug(err.Error())
 		return false, errors.New("Failed to get Windows release id")
```

Comment:
This is exactly the same error returned in this case and before, which could make it harder to figure out which of the 2 functions failed. This one could be changed to "Failed to parse Windows release id: %s", stdOut